lxd/instance_post.go

```diff
@@ -21,10 +21,13 @@ import (
 	"github.com/lxc/lxd/lxd/project"
```

Comment:
The commit message seems to stop halfway through a sentence:

> When supplying --target use the @ character followed by a cluster group name like it is already implemented for .

Reply:
Uups, that happened during the rebase. Updated in the respective commit.